src/datahike/tools.cljc

```diff
@@ -21,3 +23,10 @@
 (defn get-time []
   #?(:clj (java.util.Date.)
      :cljs (js/Date.)))
+
+(defmacro raise [& fragments]
```

Comment:
What differences are between cljs and clj in timbre?

Reply:
afaik timbre abstracts away the differences. I think in clj it logs via your used logging-impl and in JS it uses console.log/error or something like that.